xonsh/foreign_shells.py

```diff
@@ -584,7 +584,7 @@ def load_foreign_aliases(shells=None, config=None, issue_warning=True):
             shaliases = {} if shaliases is None else shaliases
             for alias in set(shaliases) & set(xonsh_aliases):
                 del shaliases[alias]
-                if builtins.__xonsh_env__.get('XONSH_DEBUG'):
+                if builtins.__xonsh_env__.get('XONSH_DEBUG', False) > 1:
```

Comment:
You shouldn't need the `False` part here. The env will fill in with its own default.